buildspecs/j9.flags

```diff
@@ -972,6 +972,10 @@ all data is stored in their own region. It additionaly reduces footprint, mainly
 			<require flag="interp_gpHandler"/>
 		</requires>
 	</flag>
+	<flag id="interp_sigUsr2">
```

Comment:
While you're here, can you fix the description for `interp_sigQuitThread`, please? There should be a space before "signal".